index.html

```diff
@@ -8,23 +8,16 @@
     <title>Mobile Portoflio</title>
 </head>
 <body>
-    <div class="container">
         <header>
-            <h1 class="logo">Ngala</h1>
-            <menu>
-                <ul class="menu">
-                    <li class="mylist"><a href="#">Portoflio</a></li>
-                    <li class="mylist"><a href="#">About</a></li>
-                    <li class="mylist"><a href="#">Contact</a></li>
-                </ul>
-            </menu>
+            <a class="logo" href="#">Ngala</a>
             <nav>
                 <div class="mybar"></div>
                 <div class="mybar"></div>
                 <div class="mybar"></div>
                 
             </nav>
         </header>
+        <main>
         <section>
         <div class="background_image">
             <h1 class="header">I’m Ngala Mac
```

Comment:
- [ ] While you make a great effort 😃🚀I noticed that, at the bottom of your design, you're missing the curved border bottom effect as seen in the Figma design. , so please 😊Your design needs to look exactly like the design so kindly implement this in your design as well 😉💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|  ![image](https://github.com/Graceshiks/Portfolio-MobileFirst/assets/83360636/1294f75b-00c5-4a3c-b393-3842f15e5408)  |    ![image](https://github.com/ngalamac/mobile-portfolio/assets/83360636/7f4d3afb-4baf-4a4c-b0d3-4e2ab00cafcb)  |